code/game/objects/items/devices/illegalradio.dm

```diff
@@ -148,15 +160,42 @@
 	dat = jointext(dat,"") //Optimize BYOND's shittiness by making "dat" actually a list of strings and join it all together afterwards! Yes, I'm serious, this is actually a big deal
 	return dat
 
-/obj/item/device/illegalradio/proc/generate_buyer_menu(mob/user)
+	
+	
+/obj/item/device/illegalradio/proc/generate_delivery_menu(mob/user, var/datum/black_market_item/item)
+	var/dat = list()
+	dat += "<B><font size=5>["The Black Market"]</font></B><BR>"
+	dat += "<B>Please pay for an available delivery option.</B><BR>"
+	dat += "<B>You are purchasing the [item.name].</B><BR>"
+	dat += {"Cash stored: [src.money_stored]<BR><BR>"}
```

Comment:
brackets for a single line? also you could just do the above 4 lines with brackets.